strapi/strapi
# Features/menu logo

### What does it do?

- It adds **project settings** in the Strapi admin UI under `settings/overview` (for super-admins)
- It adds a **menu-logo** project setting that allows to change the main nav menu logo directly from the UI

### How to test it?

- Navigate to the `settings/overview` page

![Screen Shot 2022-04-26 at 11 46 42](https://user-images.githubusercontent.com/89356961/165272668-8eec4daf-34ad-4e36-9f0a-4d956b34ca9b.png)

- Pick a logo

![Screen Shot 2022-04-26 at 11 47 58](https://user-images.githubusercontent.com/89356961/165272893-258ff091-f85e-40c5-86a1-a9df2a23fe89.png)

- See it applyed

![Screen Shot 2022-04-26 at 11 51 15](https://user-images.githubusercontent.com/89356961/165273600-268cac21-c49f-427b-aa0e-eb83d909c5ba.png)
